Library/Homebrew/requirements/perl_requirement.rb

```diff
@@ -0,0 +1,41 @@
+require "rubygems"
+
+class PerlRequirement < Requirement
+  fatal true
+  default_formula "perl"
+
+  def initialize(tags)
+    @version = tags.shift if /(\d.\d{1,2})/ === tags.first
+    raise "PerlRequirement requires a version!" unless @version
+    super
+  end
+
+  def perl_version
+    @perl_version ||= Utils.popen_read("perl", "--version")[/(\d\.\d(\d)?)/, 1]
+  end
+
+  satisfy :build_env => false do
+    next unless which "perl"
+    version = perl_version
+    next unless version
+    Version.new(version.to_s) >= Version.new(@version)
+  end
+
+  env do
+    if Gem::Version.new(perl_version) >= Gem::Version.new(@version)
```

Comment:
I mean just `if Version.new(perl_version) >= Version.new(@version)`. The `Version` is from `version.rb`. So you don't need to `require "rubygems"`.